.github/workflows/mirror-sourcebot-private.yml

```
@@ -0,0 +1,56 @@
+
+    steps:
+      - name: Check out complete repository history
+        uses: actions/checkout@v7
+        with:
+          fetch-depth: 0
+          fetch-tags: true
+          persist-credentials: false
+
+      - name: Create mirror installation token
+        id: app-token
+        uses: actions/create-github-app-token@v3
```

actionlint (1.7.12)

[error] missing input "app-id" which is required by action "actions/create-github-app-token@v3". all required inputs are "app-id", "private-key"

(action)